algorithmconfig.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022-, Semiotic AI, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+from autoagora_agents import algorithm_ingredient
+
+
+@algorithm_ingredient.config
+def config():
+    seed = 0
```

Comment:
I really think the seed should be also set for entities, distributor etc.

Reply:
There's no stochasticity in the env. I could add a `seed` field if you'd like, but it'd be unused.

Reply:
4339aa8b12eb6350ba3780bea583ab7429462a03

Reply:
yeah, this env is fully deterministic. But it can change with other envs. Anyway, I guess we are aligned.